api/authenticator.py

```diff
@@ -649,12 +652,10 @@ def decode_bearer_token(self, token):
         return (provider_name, token)
 
     def create_authentication_document(self):
-        """Create the OPDS authentication document to be used when
+        """Create the Authentication For OPDS document to be used when
         a request comes in with no authentication.
         """
-        base_opds_document = Configuration.base_opds_authentication_document()
-
-        links = {}
+        links = []
```

Comment:
Links are now objects stored in a list with `rel` as one of the properties of the link object, analogous to HTML <a> tags.